### DISCUSSION TITLE
Got installed and running on Ubuntu 24.04 LTS with a small workaround

### DISCUSSION THREAD

**Discussion by cj3029412** [bluesky-social/pds#67] [Ideas]
I have got installed pds and made it running on Ubuntu 24.04 LTS with a small workaround inserting the lines below in install.sh:

> elif "${DISTRIB_CODENAME}" == "noble" ; then
>   SUPPORTED_OS="true"
>   echo "* Detected supported distribution Ubuntu 24.04 LTS"
> fi

At https://nekohana.blue/xrpc/_health. Just I wanted to say hello and thank you.

**Comment by bnewbold** [comment]
Thanks for the update. We'll probably stick with 22.04 a bit longer, but good to know things work on the next LTS release!

**Comment by SuperMohit** [comment]
Hey I am unable to make Post. Is it because of SMPT setup?

**Comment by ssarhangi** [comment]
Thank you for this!
